### ISSUE TITLE
Reply to sent email does not show in Inbox

### ISSUE THREAD

**Issue by JpF4E** [protz/thunderbird-conversations#1247]
* **Thunderbird version:** 52.3.0
* **Thunderbird Conversation version:** 2.13.3
* **Operating System:** Ubuntu

**What did you do?**
Considering I am User B, the message flow was the following:

- Email 1 (Forwarded from another email that didn't include me) From: User A, To: User B, C, D, Cc: User E;
- Email 2 (Reply to the Forwarded email) From: User B, To: User A;
- Email 3 (Reply to own email (Email 1) with different users) From: User A, To: User B, C, D, Cc: User E;

**What did you expect to happen?**
I expected a tree with the 3 emails appeared.

**What actually happened?**
A tree with 2 emails appeared. On the list, Email 1 and Email 3 appeared (in a tree). However If I click the tree or if I click the last email I received (Email 3), the contents below (where you are able to expand and collapse emails of the tree) show: Email 1 and Email 2.
So basically, if I click Email 3, it is shown Email 2 and I was never able to read the contents of Email 3. I disabled the addon, all good. When I re-enabled it, it shows my last sent message instead of the latest email I received (whose contents I can never access).

**Comment by Standard8** [comment]
Please can you go to Tools -> Developer Tools -> Error Console, and have that open when you select the email thread. See if there are any errors that appear when selecting the email thread.

**Comment by Standard8** [comment]
Unfortunately we never got more details about this. On re-reading my suspicion would be some of the de-duplicating code, not sure why though.

Not had other reports about this, so I'll close it. If it is still happening with the latest Thunderbird & Conversations, please feel free to re-open or file a new issue.

**Comment by pfheatwole** [comment]
I just experienced a similar issue. Short answer: I had to rebuild the global messages database for my reply to show in the conversation view.

Long answer:

I received a group email from "User A" and replied to "User A" only by clicking the "Reply" button attached to message in conversation view. After I sent my reply, in the messages pane the original email showed the "replied to" icon, but no tree with my newly sent message. Clicking on the original message also did not show my reply in the conversation view.

I tried exiting Thunderbird and waiting a few minutes in case it was a synchronization problem, but nothing changed. I opened the error console, and noticed the clicking on the original message logged the warning "Streaming the message because Gloda has not indexed it, this is BAD" (from [here](https://github.com/thunderbird-conversations/thunderbird-conversations/blob/f4e0a14a17de5d662ab6288eb0764d1c63bd1a24/addon/content/modules/message.js#L1032))

I rebuilt the global messages database (gloda) following the instructions at https://support.mozilla.org/en-US/kb/rebuilding-global-database. I waited for the reindex and checked again. The original message still has the "replied to" icon in the messages pane, but still no tree of messages for that conversation. However, clicking on the original message shows my reply in the conversation view. I'm not sure if the messages pane is supposed to show a tree in this case since I did a "reply" instead of "reply all", but the fact that my reply appears in the conversation view makes me suspect it is.

@Standard8: I'm not bothered by this behaviour, it just struck me as unexpected; if this is a bug and you'd like me to run any diagnostics, I'm happy to do what I can.
